src/account-info/sign-up.js

```diff
@@ -44,9 +45,9 @@ class SignUp extends Component {
           </div>
           <div>
             <p>Already have an account?</p>
-            <Link to='/sign-in'>
-                <button>Sign In</button>
-            </Link>
+                <button onClick={() => goToRoute(LOG_IN)}>
+                  Sign In
+                </button>
```

Comment:
whats the advantage of `goToRoute` over `<Link />`?

Reply:
oh... I found goToRoute is a function you made. keep in mind that if you are working with other devs who are familiar with react-router they immediately know what you are doing with a `<Link />` but will have to go searching for the custom implementation of  `goToRoute`. 

That "cognitive overhead" means slower grumpier co-devs. :-)